src/schema/re_frame.cljs

```diff
@@ -5,6 +5,16 @@
 (def ^:private ?cofx
   [:map])
 
+(def ^:private ?event-fx
+  [:maybe
+   [:map {:closed true}
+    [:db {:optional true} [:maybe map?]]
```

Comment:
In our production code, is there a case where the `db` is nil? I thought that wasn't possible. If that's the case, then it would be better to drop the `maybe` schema here.